### DISCUSSION TITLE
AsusFX504 - 0.7.9 - HDMI and Audio EFI variants

### DISCUSSION THREAD

**Discussion by RobyRew** [RobyRew/ASUS-FX504GE-Hackintosh_OpenCore#12] [General]
### News / Fixes:
- Deleted: `enable-hdmi20` `disable-hdmi-patches` `disable-hdmi-patches` `enable-hdmi-dividers-fix` property from iGPU Properties, as are useless and LSPCON does the job.
- Universal Control is working fine.
- Added on EFI > OC > ACPI > `SSDT-USBMap.aml`(Custom) and deleted the kext since SSDT is the native edition.
- Added DeviceProperties > Delete > `PciRoot(0x0)/Pci(0x1F,0x3)` for audio card layout-id (If you want to have audio over HDMI move the property under DeviceProperties > Add >)
- Added `BlueToolFixup.kext` and `FeatureUnlock.kext` for better compatibility of Bluetooth on Monterrey.
- Added `BrightnessKeys.kext` and now Brightness keys works natively with `fn` + `F7`/`F8`
- Deleted `USBMap.kext` obviously.

### Known Issues:
- AsusFX504-RELEASE_BuggyHDMI_HQ_Audio.zip will give you perfect audio on jack3.5, microphone, speakers or bluetooth earbuds.
- AsusFX504-RELEASE_NoAudio_JustOverBluetooth.zip will give you perfect HDMI output lid of internal display, etc. bluetooth earbuds also works, but jack3.5, microphone and speakers are not working at all.

**ISSUE MAY BE RELATED WITH WhateverGreen OR AppleALC I DON'T KNOW.** 

### Important Note:
- Change the [Airportitlwm.kext](https://github.com/OpenIntelWireless/itlwm/releases) version for your macOS version, EFI contains the kext for Monterrey:

<hr /><em>This discussion was created from the release <a href='https://github.com/RobyRew/ASUS-FX504GE-Hackintosh_OpenCore/releases/tag/0.7.9'>AsusFX504 - 0.7.4 - HDMI and Audio EFI variants</a>.</em>

**Comment by Uzurka** [comment]
Hey and thanks for ur work, i have a similar laptop with some variants (i5 instead of i7)
Still looking for a all working speakers headset and HDMI audio working. Have you found something ? 
By the way, is an update to the latest OC planned? I use my own EFI with some of your settings atm, and i'll continue, i'm looking or waiting for an HDMI audio fix :)
